.gitignore

```diff
@@ -32,3 +32,6 @@ resotocore/tools/Assets
 out/
 *.iml
 **/.hypothesis/
+plugins/api_sandboxes/*.ipynb
```

Comment:
Why plugins?
I would opt for `.sandbox/` - the dot for hiding this directory and everything in it (no matter which extension).